### ISSUE TITLE
Publish new version on npm

### ISSUE THREAD

**Issue by raivieiraadriano92** [ptomasroos/react-native-scrollable-tab-view#1019]
Please, publish a new version on npm with fix: https://github.com/PizzaLiu/react-native-scrollable-tab-view/commit/b45bde7fc11b688d0879f62dddb0018dc4355ac9

Thanks

**Comment by nitaking** [comment]
agree.

**Comment by laurentS** [comment]
For now, I've just updated my package.json with:
```diff
+    "@react-native-community/viewpager": "^1.1.7",
...
-    "react-native-scrollable-tab-view": "^0.10.0",
+    "react-native-scrollable-tab-view": "https://github.com/PizzaLiu/react-native-scrollable-tab-view#b45bde7fc",
```

It does the job with RN 0.59, but agree that an update would be nice.

**Comment by mlbz198** [comment]
> For now, I've just updated my package.json with:
> 
> ```diff
> +    "@react-native-community/viewpager": "^1.1.7",
> ...
> -    "react-native-scrollable-tab-view": "^0.10.0",
> +    "react-native-scrollable-tab-view": "https://github.com/PizzaLiu/react-native-scrollable-tab-view#b45bde7fc",
> ```
> 
> It does the job with RN 0.59, but agree that an update would be nice.

Thank you so much!

**Comment by ptomasroos** [comment]
Released as 1.0.0
